core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,26 +2,26 @@
 Benchmark ZStandardCompressionCodec
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 17.0.9+8-LTS on Linux 5.15.0-1050-azure
-Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
+OpenJDK 64-Bit Server VM 17.0.9+9-LTS on Linux 5.15.0-1051-azure
+AMD EPYC 7763 64-Core Processor
 Benchmark ZStandardCompressionCodec:                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 --------------------------------------------------------------------------------------------------------------------------------------
-Compression 10000 times at level 1 without buffer pool           2786           2787           2          0.0      278560.1       1.0X
-Compression 10000 times at level 2 without buffer pool           2831           2833           3          0.0      283091.0       1.0X
-Compression 10000 times at level 3 without buffer pool           2958           2959           2          0.0      295806.3       0.9X
-Compression 10000 times at level 1 with buffer pool               211            214           4          0.0       21145.3      13.2X
-Compression 10000 times at level 2 with buffer pool               253            255           1          0.0       25328.1      11.0X
-Compression 10000 times at level 3 with buffer pool               370            371           1          0.0       37046.1       7.5X
+Compression 10000 times at level 1 without buffer pool            657            659           2          0.0       65716.9       1.0X
```

Comment:
A new CPU model (AMD EPYC 7763) has been added to GA, but the benchmark results look significantly different from before. Let me see if we can retry the test on an Intel CPU.

Reply:
Got it. For now, let's ignore this.

Reply:
We have enough time for Apache Spark 4.0.0. So, we can find a way later. In addition, we will have more zstd-jni update.